.gitignore

```diff
@@ -13,6 +13,7 @@ target/
 /log/
 /output/
 /scripts/
+/janusgraph-codepipelines-ci/dependency-reduced-pom.xml
```

Comment:
Can you update Maven shade plugin config in janusgraph-codepipelines-ci/pom.xml to include createDependencyReducedPom=false and then remove this exclusion?